.github/workflows/ci.yml

```diff
@@ -65,18 +69,26 @@ jobs:
           bundle install --jobs 4 --retry 3
           yarn install
 
-      - name: Set up app
-        run: |
-          bundle exec rails db:setup
-
       - name: Run linters
         run: |
           bundle exec rubocop
           bundle exec brakeman
 
+      - name: Set up app
+        env:
+          POSTGRES_HOST: localhost
```

Comment:
These changes were key to fixing the weird hostname thing. When a `container` isn't specified for these GitHub actions, this is all `localhost`.